members/C001075.yaml

```diff
@@ -48,11 +48,11 @@ contact_form:
         value: $MESSAGE
         required: true
     - javascript:
-      - name: Message
```

Comment:
Had to change the name value to 'char replace' because submissions were failing otherwise.